docker-deploy/training_template/backends/spark/spark/spark-defaults.conf

```diff
@@ -1,3 +1,4 @@
 spark.master                      spark://spark-master:7077 
 #spark.eventLog.enabled           true
 #spark.eventLog.dir               hdfs://namenode:9000/spark/logs
+spark.cores.max                   4
```

Comment:
Why we need to change this to 4?

Reply:
It can be defined by `compute_core` in parties.conf, 4 is the default value.